### PR TITLE
add 4Cast fees adaptor

### PR DESCRIPTION
**Pull request by 4Cast-Studio**
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The 4cast adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees 4cast
    
    🦙 Running 4CAST adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 18 Aug 2024 05:01:39 GMT
    End Date:	Mon, 19 Aug 2024 05:01:39 GMT
    ---------------------------------------------------
    
    SOLANA 👇
    Backfill start time: 17/7/2024
    NO METHODOLOGY SPECIFIED
    Daily fees: 0
    End timestamp: 1724043698 (2024-08-19T05:01:38.000Z)

**Comment by 4Cast-Studio** [comment]
Running the test prints "Daily fees: 0" even though the API returns a valid positive value. Any idea how to fix it?

File: fees/4cast/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+interface IData {
+  dailyFees: string;
+}
+
+const endpoint = "https://www.4cast.win/api/api/platformFees";
+
+function createSolBalances(options: FetchOptions, value: string) {
+  const balances = options.createBalances();
+
+  balances.addGasToken(Number(value));
```
Comment (dtmkeng) [inline fees/4cast/index.ts]: try use 
```js
 balances.add('So11111111111111111111111111111111111111112', (Number(value) * 1e9))
 ```

**Comment by llamatester** [comment]
The 4cast adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees 4cast
    
    🦙 Running 4CAST adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 20 Aug 2024 06:06:42 GMT
    End Date:	Wed, 21 Aug 2024 06:06:42 GMT
    ---------------------------------------------------
    
    SOLANA 👇
    Backfill start time: 17/7/2024
    NO METHODOLOGY SPECIFIED
    Daily fees: 4.20 k
    End timestamp: 1724220401 (2024-08-21T06:06:41.000Z)

**Comment by 4Cast-Studio** [comment]
I have made the requested changes. It works correctly now. Thanks.

**Pull request merged**